CMakeLists.txt

```
@@ -41,6 +41,10 @@ set(CMAKE_C_STANDARD_REQUIRED ON)
 set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 set(CMAKE_C_VISIBILITY_PRESET hidden)
 set(OQS_VERSION_TEXT "0.11.1-dev")
+set(OQS_VERSION_MAJOR 0)
+set(OQS_VERSION_MINOR 11)
+set(OQS_VERSION_PATCH 1)
+set(OQS_VERSION_PRE_RELEASE "-dev")
```

What about generating `OQS_VERSION_TEXT` automatically based on the other variables? This reduces the risk of a manual error that leads to the version variables not matching.
```suggestion
set(OQS_VERSION_MAJOR 0)
set(OQS_VERSION_MINOR 11)
set(OQS_VERSION_PATCH 1)
set(OQS_VERSION_PRE_RELEASE "-dev")
set(OQS_VERSION_TEXT "${OQS_VERSION_MAJOR}.${OQS_VERSION_MINOR}.${OQS_VERSION_PATCH}${OQS_VERSION_PRE_RELEASE}")
```